### PR TITLE
Docs/coverage

### PR DESCRIPTION
**Pull request by midmiddleton**

### CODE REVIEW

File: docs/content/tooling/coverage.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import {InfoIcon} from '@primer/styled-octicons'
+import InlineCode from '@primer/gatsby-theme-doctocat/src/components/inline-code'
+import {Link} from 'gatsby';
+
+When writing tests it can often prove difficult to ensure we have sufficient coverage across all our code. 
+In this section we will look at using a coverage report to allow us to determine if further testing is required.
+
+## Generating a report
+
+It is first important to ensure the tests are passing and to a high standard. read the <Link to="/tooling/testing">testing</Link> section more infomation
```
Comment (Smudge3806) [inline docs/content/tooling/coverage.mdx:17]:
```suggestion
It is first important to ensure the tests are passing and to a high standard. Read the <Link to="/tooling/testing">testing</Link> section more information.
```

File: docs/content/tooling/coverage.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Viewing a report
+
+To see the results of the report you will need to locate the index.html file that will have been generated and placed in the coverage folder and open this in your browser.
+
+## Understanding a report.
+
+Once open you will see a breakdown of the project and an indicator as a percentage of how good the test coverage is. We strive to maintain a code coverage of at least 80%.
+
+Where codecoverage is not up to par we can click individual sections and drill down further so we are able to easily see which aspects of the code require further testing.
```
Comment (Smudge3806) [inline docs/content/tooling/coverage.mdx:33]:
```suggestion
Where code coverage is not up to that standard we can click individual sections and drill down further so we are able to easily see which aspects of the code require further testing.
```

File: docs/content/tooling/coverage.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Coverage
+description: How to generate a code coverage report
+status: Stable
+source: https://github.com/xedi/sdk-js/tree/master/testing
```
Comment (Smudge3806) [inline docs/content/tooling/coverage.mdx:5]: Is this necessary? I'm torn. I see the value but at the same time it feels out of place... /shrug

File: docs/content/tooling/coverage.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+---
+import {Text, Flash, BorderBox} from '@primer/components'
```
Comment (Smudge3806) [inline docs/content/tooling/coverage.mdx:7]:
```suggestion
import {Flash} from '@primer/components'
```

File: docs/content/tooling/coverage.mdx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+When writing tests it can often prove difficult to ensure we have sufficient coverage across all our code. 
+In this section we will look at using a coverage report to allow us to determine if further testing is required.
+
+## Generating a report
+
+It is first important to ensure the tests are passing and to a high standard. read the <Link to="/tooling/testing">testing</Link> section more infomation
+Once that is done a coverage report can be generated by simply running: <InlineCode>npm run coverage</InlineCode>.
+
+<Flash scheme="yellow">
```
Comment (Smudge3806) [inline docs/content/tooling/coverage.mdx:20]: I would be tempted to stick with the default theme.

Blue = info
Yellow = warning
Red = danger

These are the generally accepted semantics of those colours

Comment (midmiddleton) [reply docs/content/tooling/coverage.mdx:20]: I saw this as a warning that the code coverage folder would need to be ignored so the choice of yellow was intentional. Are you thinking this is more an info flash?

**Pull request closed without merging**